relay-server/src/actors/events.rs

```diff
@@ -170,6 +170,21 @@ impl ProcessingError {
     }
 }
 
+/// Returns the most relevant rate limit.
+///
+/// The most relevant rate limit is the longest rate limit for events and if there is no
+/// rate limit for events then the longest rate limit for anything else
+fn most_relevant(rate_limits: &RateLimits) -> Option<&RateLimit> {
```

Comment:
This function actually returns an error event rate limit, should we rename this? Once we implement categories properly, we can revert this to use `longest` again.

Reply:
The function itself doesn't care about what the rate_limits represent or will be used for, it just looks at sequence of rate limits and picks up the longest rate limit that would apply to events (errors or security). I think that `most_relevant` is pleasantly ambiguous :-) , but I'm open for better suggestions.